src/Utilities/src/MoistThermodynamics.jl

```diff
@@ -216,32 +217,37 @@ function latent_heat_generic(T, LH_0, cp_diff)
 
 end
 
+abstract type Phase end
+struct Liquid <: Phase end
+struct Ice <: Phase end
+struct Generic <: Phase end
+
 """
-    sat_vapor_press_liquid(T)
+    sat_vapor_press(T, Liquid())
 
 Return the saturation vapor pressure over a plane liquid surface at
 temperature `T`.
 """
-function sat_vapor_press_liquid(T)
+function sat_vapor_press(T, phase::Liquid)
 
-    return sat_vapor_press_generic(T, LH_v0, cp_v - cp_l)
+    return sat_vapor_press(T, LH_v0, cp_v - cp_l, Generic())
```

Comment:
replace these one-liners with inline form, and you don't need an argument name (unless you're passing it to another function:
```
saturation_vapor_pressure(temp, ::Liquid) = saturation_vapor_pressure(temp, LH_v0, cp_v - cp_l)
```